src/Microsoft.CodeAnalysis.Testing/Directory.Build.props

```diff
@@ -88,7 +88,7 @@
   <!-- Public API Analyzer configuration -->
   <ItemGroup>
     <PackageReference Include="Roslyn.Diagnostics.Analyzers" Version="$(RoslynDiagnosticsAnalyzersVersion)" PrivateAssets="all" />
-
+    <PackageReference Include="System.Net.Http" Version="$(SystemNetHttpVersion)" PrivateAssets="all" />
```

Comment:
Can we also include `ExcludeAssets="all"` to make sure the build doesn't reference anything from this library (since the library is not included in the declared dependencies of the package)?